From: Thomas Renninger <trenn@suse.de>
Subject: Add more HP machines to use pci=use_crs by default
References: bnc#578572
Patch-Mainline: yes and no, see below

This patch will never show up in mainline. BIOSes after a
specific date will use pci=use_crs by default.
So functionality is there and patches do not need to be
maintained in future kernel versions.

---
 arch/x86/pci/acpi.c |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

Index: linux-2.6.32-SLE11-SP1/arch/x86/pci/acpi.c
===================================================================
--- linux-2.6.32-SLE11-SP1.orig/arch/x86/pci/acpi.c
+++ linux-2.6.32-SLE11-SP1/arch/x86/pci/acpi.c
@@ -40,6 +40,22 @@ static const struct dmi_system_id pci_us
 			DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant DL980 G7"),
 		},
 	},
+	{
+		.callback = set_use_crs,
+		.ident = "HP ProLiant DL370 G6",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Hewlett-Packard"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant DL370 G6"),
+		},
+	},
+	{
+		.callback = set_use_crs,
+		.ident = "HP ProLiant ML370 G6",
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Hewlett-Packard"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "ProLiant ML370 G6"),
+		},
+	},
 	{}
 };
 
